signin/js/handle-event/handle-submit.js

```diff
@@ -0,0 +1,36 @@
+import displayEmailError from '../ui/display-email-error.js'
+import displayPasswordError from '../ui/display-password-error.js'
+
+export default function handleSubmit(e) {
+  e.preventDefault()
+
+  const email = document.getElementById('signIn-email')
+  const password = document.getElementById('signIn-password')
+  const form = document.getElementById('signIn-form')
+
+  const emailValue = email.value
+  const passwordValue = password.value
```

Comment:
@ugaugaugaugaugauga 

HTML selector들은 `[selectorName]El` 형태로 두거나 `$[selectorName]` 형태로 변수에 관리하는게 컨벤션입니다. 그리고 실제 데이터들은 해당 데이터를 표현하는 변수를 사용하죠.

```suggestion
  const $email = document.getElementById('signIn-email')
  const $password = document.getElementById('signIn-password')
  const $form = document.getElementById('signIn-form')
 
   //또는
  const emailEl = document.getElementById('signIn-email')
  const passwordEl = document.getElementById('signIn-password')
  const formEl = document.getElementById('signIn-form')

  const email = email.value
  const password = password.value
```